### DISCUSSION TITLE
453 - You currently have access to Twitter API v2 endpoints and limited v1.1 endpoints only.

### DISCUSSION THREAD

**Discussion by devkarim** [tweepy/tweepy#2118] [Questions] [answered]
Hi there!

I get this error every time I search via Tweepy:

`453 - You currently have access to Twitter API v2 endpoints and limited v1.1 endpoints only. If you need access to this endpoint, you may need a different access level.`

I'm not sure why I'm getting this. I'm subscribed to the free tier. I worked on a different project in a few weeks ago and it worked with no issues. I tried to search for this error but Twitter docs were so confusing. Sometimes it says it is allowed to use the search endpoint (Standard v1.1) and sometimes it says I'm only allowed to post up to 1500 tweets only with no other endpoints. Any clue?

Here is where it says what Standard v1.1 should include (it says free there): https://developer.twitter.com/en/docs/twitter-api/v1
Here is where "Access to standard v1.1" says: https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api#v2-access-leve

Code:
```python
import tweepy

consumer_key = ""
consumer_secret = ""
access_token = ""
access_token_secret = ""

auth = tweepy.OAuthHandler(consumer_key, consumer_secret)
auth.set_access_token(access_token, access_token_secret)

api = tweepy.API(auth)
api.verify_credentials()

tweets = tweepy.Cursor(api.search_tweets, q="#NBAtwitter", count=10).items()
for tweet in tweets:
    print(tweet.user.id)
```

**Comment by Harmon758** [comment, marked as answer]
Twitter API v1.1 is not available with Free access.
https://twittercommunity.com/t/announcing-new-access-tiers-for-the-twitter-api/188728

**Comment by meowsome** [comment]
I am confused by this. On the page I linked to below, it notes that the "Free" tier has "Only Media Upload and statuses/update and Login With Twitter" via the v1.1 API. So, it appears that the Free tier does have access to some parts of v1.1, particularly the media upload? I used to be able to use `update_status_with_media` just fine up until May 18th, but now it is giving me the same error as OP. 

https://developer.twitter.com/en/docs/twitter-api/getting-started/about-twitter-api

**Reply by Harmon758** [reply]
Yes, Twitter API v1.1's media upload endpoints are available with Free access because Twitter API v2 doesn't support uploading media yet.
